plugins/module_utils/network/sonic/config/l3_interfaces/l3_interfaces.py

```diff
@@ -468,6 +500,12 @@ def get_delete_l3_interfaces_requests(self, want, have):
                 if have_ipv6_enabled and ipv6_enabled is not None:
                     request = {"path": ipv6_enabled_url.format(intf_name=name, sub_intf_name=sub_intf), "method": DELETE}
                     requests.append(request)
+                if have_ipv6_autoconf and ipv6_autoconf is not None:
+                    request = {"path": ipv6_autoconf_url.format(intf_name=name, sub_intf_name=sub_intf), "method": DELETE}
+                    requests.append(request)
+                if have_ipv6_dad and ipv6_dad is not None:
```

Comment:
Please delete only when the specified configuration exists.
```suggestion
                if have_ipv6_dad and ipv6_dad == have_ipv6_dad:
```

Reply:
Done